### PR TITLE
Update conanfile.py for flatbuffers v1.8.0

### PR DESCRIPTION
**Pull request by ndesai**

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -3,33 +3,44 @@
...
     name = "FlatBuffers"
-    version = "1.5.0-dm1"
-    license = "BSD 2-Clause"
+    license = "https://github.com/google/flatbuffers/blob/master/LICENSE.txt"
+    description = "FlatBuffers is an efficient cross platform serialization library for games and other memory constrained apps. It allows you to directly access serialized data without unpacking/parsing it first, while still having great forwards/backwards compatibility."
     url = "https://github.com/ess-dmsc/conan-flatbuffers"
     settings = "os", "compiler", "build_type", "arch"
     generators = "cmake"
-    options = {"shared": [True, False]}
+    options = { "shared": [True, False] }
```
Comment (amues) [inline conanfile.py]: Would you mind removing the spaces here? Flake8 doesn't like them.

File: conanfile.py [hunk 2]
```diff
@@ -47,18 +58,18 @@ def build(self):
...
         self.copy("flathash", dst="bin",
-                  src="flatbuffers-1.5.0/build", keep_path=False)
+                  src=self.build_dir, keep_path=False)
         self.copy("*.h", dst="include/flatbuffers",
-                  src="flatbuffers-1.5.0/include/flatbuffers")
+                  src="%s/include/flatbuffers" % self.folder_name)
         self.copy("*.a", dst="lib",
-                  src="flatbuffers-1.5.0/build", keep_path=False)
+                  src=self.build_dir, keep_path=False)
         self.copy("*.so", dst="lib",
```
Comment (amues) [inline conanfile.py]: FlatBuffers now generates a library called lib flatbuffers.so.1.8.0 and links named flatbuffers.so.1 and flatbuffers.so. Can you change the `package` code to copy *.so* and *.dylib* instead?

**Comment by ndesai** [comment]
Still a little new at pull requests (coming from a Gerrit background), is it okay that I rebased to master?

**Comment by amues** [comment]
Hi @ndesai, sorry for the late reply. Thanks for the changes, they look good! I'll merge them.

**Pull request merged**